modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js

```diff
@@ -40,8 +40,9 @@ describe('DatePicker', () => {
 		render(<DatePicker label="Date picker" />);
 
 		const allByText = screen.getAllByText('Date picker');
-		expect(allByText).toHaveLength(1);
+		expect(allByText).toHaveLength(2);
```

Comment:
why do we have two now? :thinking:

Reply:
Is it because of the hidden one?

Reply:
yes, i actually came back to what was before selena sent the fix, the test was exactly like this

if you see the tests from the other field types they are doing the exactly same thing

Reply:
it's a span hidden that is used for accessibility

Reply:
So it LGTM! Just forward ir, if failures are not related.

Reply:
thanks!